src/view/nova/themes.ts

```diff
@@ -0,0 +1,184 @@
+import {createTheme} from './lib/theme'
+
+const BLUE_HUE = 210
+const GRAYSCALE_SATURATION = 12
+
+export const palette = {
+  white: '#FFFFFF',
+
+  /**
+   * Mathematical scale of grays, from lightest to darkest, all based on the
+   * primary blue color
+   */
+  gray1: `hsl(${BLUE_HUE}, ${GRAYSCALE_SATURATION}%, 95%)`,
+  gray2: `hsl(${BLUE_HUE}, ${GRAYSCALE_SATURATION}%, 85%)`,
+  gray3: `hsl(${BLUE_HUE}, ${GRAYSCALE_SATURATION}%, 75%)`,
+  gray4: `hsl(${BLUE_HUE}, ${GRAYSCALE_SATURATION}%, 30%)`,
+  gray5: `hsl(${BLUE_HUE} ${GRAYSCALE_SATURATION}%, 20%)`,
+  gray6: `hsl(${BLUE_HUE}, ${GRAYSCALE_SATURATION}%, 10%)`,
+  black: `hsl(${BLUE_HUE}, ${GRAYSCALE_SATURATION}%, 5%)`,
+
+  blue: `hsl(${BLUE_HUE}, 100%, 53%)`,
+  green: '#54D469',
+  red: '#FB4566',
+}
+
+export const light = createTheme({
+  tokens: {
+    space: {
+      xxs: 2,
+      xs: 4,
+      s: 8,
+      m: 12,
+      l: 18,
+      xl: 24,
+      xxl: 32,
+    },
+    color: {
+      primary: palette.blue,
+      l0: palette.white,
+      l1: palette.gray1,
+      l2: palette.gray2,
+      l3: palette.gray3,
+      l4: palette.gray4,
+      l5: palette.gray5,
+      l6: palette.gray6,
+      l7: palette.black,
+    },
```

Comment:
My thinking on naming convention here is that `l0` is always the "background" and `l7` is always the "foreground", for both light/dark themes. There are 7 because there are 6 grays in our designs rn, and we have a white and a black, and I thought zero-indexed made sense.

Extremely open to ideas here, there are a lot of solutions out there for this.